Epam.TestAutomation/Epam.TestAutomation.Tests/SeleniumBasicTests.cs

```diff
@@ -0,0 +1,114 @@
+﻿using NUnit.Framework;
+using OpenQA.Selenium.Interactions;
+using OpenQA.Selenium.Support.UI;
+using OpenQA.Selenium;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Epam.TestAutomation.Core.Browser;
+using Epam.TestAutomation.Helper;
+using Epam.TestAutomation.Utilities.ElementUtils;
+using Epam.TestAutomation.Core.WebDriverManager;
+using Epam.TestAutomation.Web.PageObjects.Pages;
+
+namespace Epam.TestAutomation.Tests
+{
+    [TestFixture]
+    public class Selenium_AdvancedTest
+    {
+        [SetUp]
+        public void Setup()
+        {
+            WebDriverManager.Initialize();
+        }
+
+        [TearDown]
+        public void TearDown()
+        {
+            WebDriverManager.Quit();
+        }
+
+        [Test]
+        public void SearchAndVerifyResults()
+        {
+            var loginPage = new LoginPage(WebDriverManager.Driver);
+            loginPage.PerformSearchAndVerifyResults();
```

Comment:
It is not a goof idea to create the method in page object for steps for test. Because it is not clear what happens during test. Move all steps from PageObject to the test.